projects/packages/jitm/src/js/jetpack-jitm.js

```diff
@@ -90,6 +106,7 @@ jQuery( document ).ready( function ( $ ) {
 					( ajaxAction ? 'data-ajax-action="' + ajaxAction + '"' : '' ) +
 					'>' +
 					envelope.CTA.message +
+					(externalLink ? EXTERNAL_LINK_ICON : '')
```

Comment:
In line with the rest of the file, I think there should be spaces here?

```suggestion
					( externalLink ? EXTERNAL_LINK_ICON : '' )
```